src/components/editor/app-bar/app-bar.tsx

```diff
@@ -16,7 +16,11 @@ import { EditorViewMode } from './editor-view-mode'
 import { HelpButton } from './help-button/help-button'
 import { NavbarBranding } from './navbar-branding'
 
-export const AppBar: React.FC = () => {
+export interface AppBarProps {
+  showEditorButtons: boolean
+}
+
+export const AppBar: React.FC<AppBarProps> = ({ showEditorButtons }) => {
```

Comment:
I don't know if it's a good idea to use one bar for two views. Especially if one of them is having less buttons. Maybe we should create a second component for that?

Reply:
As it is the same bar but for another purpose I quite like Eriks solution here… Two bars would make it hard to maintain. I think it's okay like this. One could argue a comment might improve the understanding of the prop, but I think it's fine like this.

Reply:
Okay but then I would rather use an enum than a boolean.

Reply:
You mean something like AppBarMode?

Reply:
Yes. I mean it's @ErikMichelson s decision.. But imho an enum would be better because then the purpose of this argument is clearer.

Reply:
Hm, an enum implies more than to modes and that could be the point where different bars are a good idea… 🤔

Reply:
I don't know which features else could be optional in the bar but not-editor specific. But on the other side I can't predict the future. And changing that into an enum is a rather simple task. I'll do that.

Reply:
Done in a10ebd4